backend/main.go

```diff
@@ -450,6 +451,8 @@ func main() {
 			r.Get("/assets/{project_id}/{hash_val}", privateResolver.AssetHandler)
 			r.Get("/project-token/{project_id}", privateResolver.ProjectJWTHandler)
 
+			r.Get("/validate-token", privateResolver.ValidateAuthToken)
```

Comment:
And endpoint to validate token. This is currently used when PasswordAuth client on the frontend is initialized.
The intention is to assume the token is valid initially but validate in the background and log user out if we find
out token is invalid. This might be redundant because when a private graph request fails with 401, the app signs
the user out anyway and the same "effect" is achieved.

This however is prudent.